workflows/prognostic_c48_run/runtime/emulator.py

```diff
@@ -33,7 +25,7 @@ class Config:
         online: if True, the emulator will be replace fv3 physics for all
             humidities above level ``ignore_humidity_below``.
         train: if True, each timestep will be used to train the model.
-        ignore_humidity_below: see ``online``.
```

Comment:
Include this attribute and explain when it's used

Reply:
should `output_mask` also have a docstring or is that an old name for `mask_kind`?